src/tsUnionTypes/task2.ts

```diff
@@ -1,4 +1,4 @@
-function countUppercase(argument?: string | null) {
+function countUppercase(argument?: string): number | null {
   let letArray: string[] = [];
   if (typeof argument === "string") {
```

Comment:
šitą eilutę galima daug trumpiau parašyti, nes galimi tik du argument tipai - string arba undefined. tokiu atveju galima patį `argument` tikrinti kaip boolean, nebūtina tikrinti jo tipo.